types.go

```diff
@@ -583,12 +583,19 @@ type (
 		ID            string         `json:"id,omitempty"`
 		Status        string         `json:"status,omitempty"`
 		Intent        string         `json:"intent,omitempty"`
+		Payer         *OrderPayer    `json:"payer,omitempty"`
```

Comment:
could we use the PayerWithNameAndPhone struct here? why do we need to add a new struct? or am i missing something?

from [this](https://developer.paypal.com/docs/api/orders/v2/#definition-payer) it seems like payerWithNameAndPhonehas all we want right?

Reply:
I believe your right.  I'll update the PR to remove the unnecessary extra struct.